client/components/GlobalControls/UserMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button } from '@blueprintjs/core';
+import { MenuDisclosureProps } from 'reakit/Menu';
+
+import { LoginData } from 'types';
+import { Avatar, Menu, MenuItem, MobileAware, MobileAwareRenderProps } from 'components';
+import { apiFetch } from 'client/utils/apiFetch';
+
+type Props = {
+	loginData: LoginData;
+};
+
+const handleLogout = () => {
+	apiFetch('/api/logout').then(() => {
+		window.location.href = '/';
+	});
+};
+
+const renderDisclosureButton = (
```

Comment:
First time I've encountered the word 'disclosure' used this way -- took me a minute to end up on the Reakit and WAI-ARIA breakdowns of the disclosure pattern. Struck me as semantically non-obvious, but hey, that's most variable names.